aten/src/ATen/native/native_functions.yaml

```diff
@@ -8609,17 +8609,26 @@
 #
 # See linalg_det as an example.
 
-- func: linalg_cholesky(Tensor self) -> Tensor
+# "_ex" stands for experimental
+- func: linalg_cholesky_ex(Tensor self, *, bool check_errors=False) -> (Tensor L, Tensor info)
   python_module: linalg
   variants: function
   dispatch:
-    CompositeExplicitAutograd: linalg_cholesky
+    CompositeExplicitAutograd: linalg_cholesky_ex
 
-- func: linalg_cholesky.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
+- func: linalg_cholesky_ex.L(Tensor self, *, bool check_errors=False, Tensor(a!) L, Tensor(b!) info) -> (Tensor(a!) L, Tensor(b!) info)
```

Comment:
.out?

Reply:
Jumping ahead of @ezyang here: could we look at making these structured? If that seems challenging we could look at doing it in a follow-up PR.

Reply:
I don't see a CPU, CUDA dispatch key here so this might be an alias style kernel that may not work with structured. You should add this to the binder of cases to solve with the transpiler.

Reply:
Is `CPU, CUDA` dispatch key preferred over `CompositeExplicitAutograd` if the dispatch stubs are used?

I haven't looked at porting to structured in detail, so I can't estimate how challenging is it. I was planning to port linalg module functions to structured after the 1.9 branch cut.